OMCompiler/Compiler/SimCode/SimCodeUtil.mo

```diff
@@ -2311,33 +2311,12 @@ algorithm
           eqSystlst := listAppend(eqs,resEqs);
           tempvars := createTempVarsforCrefs(List.map(solveCr, Expression.crefExp),itempvars);
         else
-          if match (e1,e2) case (DAE.RCONST(),DAE.IFEXP()) then true; else false; end match then
```

Comment:
I missed one! :laughing: #6726

Reply:
yea, this one is even weirder
```
if match () then true else; false end match; then
  try
     /* stuff */
     b := true;
  else
     b := false;
  end try;
end if;
```
is just
```
b := match ()
   case ()
      /* stuff */
   then true;
   else false;
end match;
```

Luckily this whole section is nonsense and i could remove all of it :)